jenkins/bootstrap.py

```diff
@@ -333,29 +334,42 @@ def node():
 
 
 def find_version():
-    """Determine and return the version of the build."""
-    version_file = 'version'
-    if os.path.isfile(version_file):
-        # e2e tests which download kubernetes use this path:
-        with open(version_file) as fp:
-            return fp.read().strip()
-
-    version_script = 'hack/lib/version.sh'
-    if os.path.isfile(version_script):
-        cmd = [
-            'bash', '-c', (
+    """Determine and return the version and git commit of the build."""
+    try:
+        # First try using kubectl
```

Comment:
hm, now that I think about this, I don't think neither the previous version of this nor this kubectl version work correctly for CI, since we haven't extracted the kubernetes tarball by this point.